CMakeLists.txt

```diff
@@ -434,7 +434,7 @@ SET(BROKEN_WCHAR_H "0")
 SET(HAVE_WCHAR_T "1")
 
 CONFIGURE_FILE( ${CMAKE_CURRENT_SOURCE_DIR}/iconvconf.h.cmake-in ${CMAKE_CURRENT_BINARY_DIR}/iconvconf.h )
-CONFIGURE_FILE( ${CMAKE_CURRENT_SOURCE_DIR}/include/iconv.h.in ${CMAKE_CURRENT_BINARY_DIR}/include/iconv.h @ONLY)
+CONFIGURE_FILE( ${CMAKE_CURRENT_SOURCE_DIR}/include/iconv.h.build.in ${CMAKE_CURRENT_BINARY_DIR}/include/iconv.h @ONLY)
```

Comment:
Why is it now called `.build.in`?

Reply:
This file is identical to `iconv.h.in` except that it adds all the export macros. Both are unmodified from the original sources. I don't exactly know how they are treated differently in the original autoconf/make scripts. I guess one is used during build stage and the other one an exported header.